kafka-commons/src/main/java/uk/gov/homeoffice/digital/sas/kafka/producer/KafkaProducerService.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.homeoffice.digital.sas.kafka.producer;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Component;
+import uk.gov.homeoffice.digital.sas.kafka.message.KafkaAction;
+import uk.gov.homeoffice.digital.sas.kafka.message.KafkaEventMessage;
+
+@Component
```

Comment:
nit: `@Service` annotation is slightly more suitable

Reply:
Actioned